data/data/openstack/common-variables.tf

```diff
@@ -0,0 +1,194 @@
+variable "auth_url" {
```

Comment:
the name doesn't describe if this is used for talking to `openstack api` or `kubernetes api`